sagemaker-huggingface/text-summarizer/huggingface-custom-text-summarizer.ipynb

```diff
@@ -0,0 +1,2626 @@
+{
```

Comment:
<p>Grammar nit - "straight forward" is a single word, "straightforward"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3095/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='43'/>